### PR TITLE
Add fnv to stdx

### PR DESCRIPTION
**Pull request by alisha17**
fnv crate example using FNV HashMap

Fix to issue #34

### CODE REVIEW

**Comment by alisha17** [comment]
Why does the build fail?

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Current revision: `stdx` 0.118.0-rc, for Rust 1.18, June 8, 2017.
 | Date and time                  | [`chrono = "0.3.1"`]       | [📖][d-chrono]      |
 | Command-line argument parsing  | [`clap = "2.24.2"`]        | [📖][d-clap]        |
 | Error handling                 | [`error-chain = "0.10.0"`] | [📖][d-error-chain] |
+| FNV hash function              | [`fnv = "1.0.3"`]          | [📖][d-fnv]         |
```
Comment (brson) [inline README.md]: Since most people won't know what "FNV" is, how about we say "Fast (FN) hashing".

File: examples/fnv.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+map.insert(3, "three");
+
+for (number, word) in map.iter() {
+        println!("Number {}: {}", number, word);
+}
+
+map.remove(&(2));
+println!("The length of HashMap is {}.", map.len());
+println!("The first element is {}.", map.get(&(1)).unwrap());
+}
```
Comment (brson) [inline examples/fnv.rs]: The inside of this function needs to be intended one level, and likewise in the markdown.

File: README.md [hunk 3]
```diff
@@ -1140,6 +1175,7 @@ copyright is owned by its contributors.
 [d-clap]: https://docs.rs/clap/2.24.2/clap/
 [d-error-chain]: https://docs.rs/error-chain/0.8.1/error_chain/
 [d-flate2]: https://docs.rs/flate2/0.2.19/flate2/
+[d-fnv]: https://doc.servo.org/fnv/
```
Comment (brson) [inline README.md]: Please make this link "https://docs.rs/fnv/1.0.3/fnv/". All docs on this page link to docs.rs

**Comment by brson** [comment]
Looks great, @alisha17. Thank you! Just a few nits to fix up.

The failing CI is a preexisting problem: https://github.com/brson/stdx/pull/26

**Comment by brson** [comment]
Ping @alisha17

**Comment by brson** [comment]
Thanks @alisha17!

**Pull request merged**